### PR TITLE
feat : about

### PR DESCRIPTION
**Pull request by Utieee**
#### What does this PR do?
-Add the About page

#### Description of Task to be completed?
- 

#### How should this be manually tested?
-npm start

#### What are the relevant Trello Card stories? [ Trello Card Item ](https://trello.com/b/8x3mo2V5/say-their-names-web)
- https://trello.com/c/zt31Unan/40-about-page

#### Pull Request Type (check one)
- [x] Feature
- [ ] Bug Fixes
- [ ] Chore

#### Developer Checklist
- [ ] New code has been scanned for ESlint issues
- [x] I have tested my code locally
- [x] I made sure that there are no other pull requests that solve for this issue
- [x] My code follows the same structure within the guidelines

### CODE REVIEW

File: src/pages/About/About.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+import React from 'react';
+import {
+  StyledAbout,
+  StyledAboutContainer
+} from './style';
+
+const About = () => (
+
+
```
Comment (Bedrock02) [inline src/pages/About/About.js:10]: extra whitespace? Remove the extra lines

File: src/pages/About/About.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
```
Comment (Bedrock02) [inline src/pages/About/About.js:1]: remove empty line

File: src/pages/About/style.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ 
+`;
+
+
+export {
+
+  StyledAbout,
+  StyledAboutContainer
+
+};
```
Comment (Bedrock02) [inline src/pages/About/style.js:60]: A lot of empty space in this file, can we remove some of the empty lines that are not needed.

Comment (Utieee) [reply src/pages/About/style.js:44]: Solved, Removed empty lines

**Comment by Utieee** [comment]
> We went with naming folders with small letters, could you change the folder name fro `About` to `about`.
> 
> Also, I noticed some huge amounts of padding on the mobile view, maybe you want to add some media queries to fix that.
> 
> Once those are fixed, it's good to go for me. Good job.

@Easybuoy Changed folder name and made use of media queries. Please check as I am fairly new to media queries. Thanks

**Pull request closed without merging**